packages/models-library/src/models_library/api_schemas_webserver/folders_v2.py

```diff
@@ -16,17 +16,33 @@ class FolderGet(OutputSchema):
     folder_id: FolderID
     parent_folder_id: FolderID | None = None
     name: str
+
     created_at: datetime
     modified_at: datetime
     trashed_at: datetime | None
+    trashed_by: Annotated[
+        GroupID | None, Field(description="The primary gid of the user who trashed")
```

Comment:
in the PR description you wrote UserID, here I see now GroupID. Maybe update the PR description?

Reply:
description was wrong. thx!